aws/resource_aws_cloudwatch_metric_alarm.go

```diff
@@ -237,8 +237,11 @@ func resourceAwsCloudWatchMetricAlarm() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{"evaluate", "ignore"}, true),
 			},
 
-			"tags": tagsSchema(),
+			"tags":     tagsSchema(),
+			"tags_all": tagsSchemaComputed(),
```

Comment:
missed during first review @bflad : the `Create` method needs to pass in the new merged tags via nested func  `getAwsCloudWatchPutMetricAlarmInput`

Reply:
Updated in https://github.com/hashicorp/terraform-provider-aws/pull/18728/commits/11eb3f1b23837bfa201117bdaa19d8354795e2e3